playbooks/create_groups.yml

```
@@ -5,7 +5,7 @@
   gather_facts: False
   tasks:
     - name: Add all cluster nodes to the hadoop-cluster group
-      always_run: yes
+#      check_mode: no
```

This shouldn't be commented out otherwise it will impact the use of the --check flag when testing
```suggestion
      check_mode: no
```